src/octoprint/util/json/encoding.py

```diff
@@ -0,0 +1,66 @@
+__license__ = "GNU Affero General Public License http://www.gnu.org/licenses/agpl.html"
+__copyright__ = "Copyright (C) 2022 The OctoPrint Project - Released under terms of the AGPLv3 License"
+
+import json
+from collections import OrderedDict
+from typing import Any, Callable, Dict
+
+from frozendict import frozendict
+
+from octoprint.util import to_unicode
+
+
+class JsonEncoding:
+    # actually OrderedDict, but typing annotation for that is only available from
```

Comment:
didn't realize it--I thought python only used patch releases for bug fixes, not new stuff.

How about this instead?

```
try:  # py3.7.{0,1} support
  from typing import OrderedDict
except ImportError:
  OrderedDict = Dict
```

Reply:
Good idea, going to do that